packages/templates/clients/websocket/python/components/ClientClass.js

```diff
@@ -22,7 +23,8 @@ export function ClientClass({ clientName, serverUrl, title, queryParams }) {
       <RegisterOutgoingProcessor />
       <HandleMessage />
       <HandleError />
-      <SendEchoMessage />
+      <SendOperation sendOperations={sendOperations} clientName={clientName} />
+      {sendOperations.length > 0 && <Send />}
```

Comment:
I know `Send` do not require any arguments, but wouldn't it be easier and cleaner to use if it actually accepts `sendOperations` as input and do `sendOperations.length > 0` internally ?